CommunityFridgeMapApi/dependencies/python/constants.py

```diff
@@ -0,0 +1,3 @@
+#Tag Class constants
+MIN_TAG_LENGTH = 3
```

Comment:
I think a constants file would be good if we had global variables that more than one class uses. But since this constant is only used in the Tag class then I think these could live within the Tag class. What do you think?

Reply:
Okay! Constants aren't used a lot in Python so I was just following the standard instructions. I agree that it seems silly to have a separate file for those two.